rules/integrations/aws_bedrock/aws_bedrock_high_resource_consumption_detection.toml

```diff
@@ -45,7 +45,7 @@ timestamp_override = "event.ingested"
 type = "esql"
 
 query = '''
-from logs-aws_bedrock.invocation-*
+from logs-aws_bedrock.invocation-* metadata _id, _version, _index
 | stats max_tokens = max(gen_ai.usage.prompt_tokens),
```

Comment:
If you look at this query it's using `stats`. Im not sure you need to add this.

Reply:
Good callout, fixed.